drivers/volume/volume.go

```diff
@@ -35,6 +37,27 @@ type Driver interface {
 
 	// Stop the driver
 	Stop() error
+
+	// ClusterPairPluginInterface Interface to pair clusters
+	ClusterPairPluginInterface
+	// MigratePluginInterface Interface to migrate data between clusters
+	MigratePluginInterface
+}
+
+// ClusterPairPluginInterface Interface to pair clusters
+type ClusterPairPluginInterface interface {
+	// Create a pair with a remote cluster
+	CreatePair(*stork_crd.ClusterPair) (string, error)
+	// Deletes a paring with a remote cluster
+	DeletePair(*stork_crd.ClusterPair) error
+}
+
+// MigratePluginInterface Interface to migrate data between clusters
+type MigratePluginInterface interface {
+	StartMigration(*stork_crd.Migration) ([]*stork_crd.VolumeInfo, error)
```

Comment:
Can you document what these interfaces do and return?

Reply:
Done